package.json

```diff
@@ -12,10 +23,18 @@
   "author": "canonical",
   "license": "ISC",
   "dependencies": {
-    "sass": "1.70.0",
-    "postcss": "8.4.31",
     "autoprefixer": "10.4.13",
+    "global": "^4.4.0",
+    "postcss": "8.4.31",
     "postcss-cli": "10.1.0",
+    "sass": "1.70.0",
     "vanilla-framework": "4.16.0"
```

Comment:
Can we try using latest vanilla verison and latest supported SASS version so that when we develop other pages we try to follow up with new changes